src/Utils/Checker/Rules/MaxAgeRule.php

```diff
@@ -75,7 +75,7 @@ public function checkRule(
             $loginParams = [];
             $loginParams['ReturnTo'] = (new HTTP())->addURLParameters((new HTTP())->getSelfURLNoQuery(), $queryParams);
 
-            $this->authenticationService->getAuthenticateUser($request, $loginParams, true);
+            $this->authenticationService->authenticate($request, $loginParams);
```

Comment:
I like this change. I think it makes it more obvious that we are just making the user authenticate rather than wanting to get the user object.